ydb/core/kqp/ut/scheme/kqp_scheme_ut.cpp

```diff
@@ -2650,10 +2675,122 @@ Y_UNIT_TEST_SUITE(KqpScheme) {
         }
     }
 
-    Y_UNIT_TEST(CreateTableWithVectorIndexPublicApi) {
+    Y_UNIT_TEST(CreateTableWithVectorIndex) {
+        NKikimrConfig::TFeatureFlags featureFlags;
+        featureFlags.SetEnableVectorIndex(true);
+        auto settings = TKikimrSettings().SetFeatureFlags(featureFlags);
+        TKikimrRunner kikimr(settings);
+        auto db = kikimr.GetTableClient();
+        auto session = db.CreateSession().GetValueSync().GetSession();
+        {
+            TString create_index_query = R"(
+                --!syntax_v1
+                CREATE TABLE `/Root/TestTable` (
+                    Key Uint64,
+                    Embedding String,
+                    PRIMARY KEY (Key),
+                    INDEX vector_idx 
+                        GLOBAL USING vector_kmeans_tree 
+                        ON (Embedding)
+                        WITH (similarity=inner_product, vector_type=float, vector_dimension=1024)
```

Comment:
To check tolower works
```suggestion
                        WITH (SIMILARITY=INNER_PRODUCT, VECTOR_TYPE=FLOAT, VECTOR_DIMENSION=1024)
```

Reply:
ed2d25e07004fc7d17c7bc55793d2f14ca38a21f